Sources/CoreData/CoreDataAccess.swift

```diff
@@ -64,6 +64,7 @@ public final class CoreDataAccess: DataAccess {
     private let storeType: StoreType
     private let container: NSPersistentContainer
     private var shouldLoadAsynchronously: Bool = false
+    private var defaultContextPolicy: NSMergePolicy = .error
```

Comment:
Feels weird to define a default value here _and_ in the `init()`; can you remove this here and just use the default value in the init? Or change this default to use `NSMergeByPropertyStoreTrumpMergePolicy`?